content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -572,6 +572,30 @@ ConfigMap updates.
 
 {{% capture discussion %}}
 
+## Auto-created ConfigMaps
+
+If BoundServiceAccountTokenVolume was enabled(requires v1.13 or later), kube-controller-manager would create ConfigMap `kube-root-ca.crt` in each namespace.
```

Comment:
This might suit a [`feature-state` shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state). If not, though, I think there should be whitespace before the “(” symbol.

Reply:
How about this (fragment):
```markdown
… kube-controller-manager will create a ConfigMap named `kube-root-ca.crt` in each namespace.
```